pkg/crawler/crawler.go

```diff
@@ -396,13 +396,24 @@ func (c *Crawler) classifyLicense(ctx context.Context) error {
 	if results.Len() > 0 {
 		for _, r := range results {
 			if licenseVal, ok := filesLicenseMap[r.Filename]; ok {
+				// skip non license detection results
+				if r.MatchType != "License" {
```

Comment:
What are the other values of `r.MatchType`? Does it make sense to make this value an ENUM?

Reply:
Intended to do the same, but don't see any enum defined by the license classifier library